app/assets/stylesheets/modules/_header.scss

```diff
@@ -12,6 +12,11 @@ header {
     font-size: 55px;
     line-height: 1;
     color: #fefeff;
+
+    a, a:hover, a:focus, a:visited {
+      color: #fefeff;
```

Comment:
Color literals like `#fefeff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Each selector in a comma sequence should be on its own single line